### DISCUSSION TITLE
GitHub Action doesn't cancel.

### DISCUSSION THREAD

**Discussion by AppimateSA** [community/community#30020] [Actions]
May anyone please help me stop a Github Action? that's stuck in a queue for hours, It can't be cancelled now how do I stop it? It was supposed to last less than a minute.

**Comment by radar-blip** [comment]
Same problem here: #30018
You can file a bug report here: [Bug-Report](https://support.github.com/contact/bug-report)

**Comment by cron410** [comment]
~~Same issue for me at https://github.com/packetworks/onetimesecret/~~

Edit: Triggering a workflow on a different repo in the same account after requesting to cancel the stuck workflow seems to have resolved the issue.
![Screen Shot 2022-08-19 at 1 32 48 PM](https://user-images.githubusercontent.com/3082899/185675426-2a156ebc-5e13-404d-869a-4a0a6d816cf3.png)
